app-modules/etno/src/Filament/Concerns/HandlesMediaUploads.php

```
@@ -87,14 +87,16 @@ protected function applyTranscriptsToItems(array $items, array $transcripts): ar
-            ->usingFileName($file->getClientOriginalName())
-            ->withCustomProperties($customProperties)
-            ->toMediaCollection($collection);
+        app(ItemRepository::class)->incrementProcessingMediaCount($item);
+
+        ProcessItemMediaUpload::dispatch(
+            $item,
+            $file->getClientOriginalPath(),
+            $file->getClientOriginalName(),
+            $file->getMimeType(),
+            $customProperties
+        );
```

PHPMD (2.15.0)

[error] Avoid using static access to class '\Metafori\Etno\Jobs\ProcessItemMediaUpload' in method 'addItemMedia'. (undefined)

(StaticAccess)